internal/staticanalysis/obfuscation/detection/generate_signals.go

```diff
@@ -0,0 +1,102 @@
+package detection
+
+import (
+	"fmt"
+	"github.com/ossf/package-analysis/internal/staticanalysis/obfuscation/parsing"
+	"github.com/ossf/package-analysis/internal/staticanalysis/obfuscation/stats"
+	"github.com/ossf/package-analysis/internal/staticanalysis/obfuscation/stringentropy"
+	"strings"
+)
+
+func getStrings(data *parsing.ParseResult) []string {
+	var extractedStrings []string
+	for _, d := range data.Literals {
+		switch d.Type {
+		case "string":
+			extractedStrings = append(extractedStrings, d.Value.(string))
+		case "float64":
```

Comment:
any reason for these cases if they're not used?

Reply:
I wanted to leave these in as a stub since eventually I might add e.g. capturing of integer codepoints which may eventually be converted into strings. Can delete the cases if you prefer, or turn it into a TODO